pages.it/common/wget.md

```diff
@@ -0,0 +1,37 @@
+# wget
+
+> Scarica file dal Web.
+> Supporta HTTP, HTTPS, FTP.
+> Maggiori informazioni: <https://www.gnu.org/software/wget>.
+
+- Scarica il contenuto dell'URL in un file (dal nome "abcd" in questo caso):
+
+`wget {{https://esempio.com/abcd}}`
+
+- Scarica il contenuto dell'URL in un file (dal nome "efgh" in questo caso):
+
+`wget -O {{efgh}} {{https://esempio.com/abcd}}`
+
+- Scarica una singola pagina web e tutte le sue risorse(script, immagini, stili, ecc..) aspettando 3 secondi tra ogni richiesta:
```

Comment:
Please add whitespace before the parenthesis. Also, consider using "aspettando 3 secondi **dopo** ogni richiesta".